e2e/benchmark_test.go

```diff
@@ -8,35 +8,49 @@ import (
 	"testing"
 
 	"github.com/Layr-Labs/eigenda-proxy/clients/standard_client"
+	"github.com/Layr-Labs/eigenda-proxy/testmatrix"
+	"github.com/stretchr/testify/require"
 )
 
 // BenchmarkPutsWithSecondary  ... Takes in an async worker count and profiles blob insertions using
 // constant blob sizes in parallel
-// TODO: test variations with V2
 func BenchmarkPutsWithSecondary(b *testing.B) {
-	testCfg := TestConfig(true, false)
-	testCfg.UseS3Caching = true
-	writeThreadCount := os.Getenv("WRITE_THREAD_COUNT")
-	threadInt, err := strconv.Atoi(writeThreadCount)
-	if err != nil {
-		panic(fmt.Errorf("Could not parse WRITE_THREAD_COUNT field %w", err))
-	}
-	testCfg.WriteThreadCount = threadInt
+	testMatrix := testmatrix.NewTestMatrix()
+	testMatrix.AddDimension(testmatrix.NewDimension(V2Enabled, []any{true, false}))
```

Comment:
not familiar with golang benchmarking tools, but will this play well with your testMatrix? Or will it just give a benchmark for the entire testMatrix time?

Reply:
I split this apart into separate tests, for v1 and v2.